src/schema/v2/partner/partner.ts

```diff
@@ -1384,18 +1384,29 @@ const Partner: GraphQLFieldConfig<void, ResolverContext> = {
       description: "The slug or ID of the Partner",
     },
   },
-  resolve: (_root, { id }, { partnerLoader }, info) => {
+  resolve: async (_root, { id }, context, info) => {
+    const { authenticatedLoaders, unauthenticatedLoaders } = context
+
     const fieldsNotRequireLoader = ["internalID"]
-    const isSlug = !/[0-9a-f]{24}/.test(id)
+    const isSlug = !/^[0-9a-f]{24}$/.test(id)
+
+    console.log("partnerLoader", authenticatedLoaders?.partnerLoader)
+    console.log(
+      "unauthenticatedLoaders.partnerLoader",
+      unauthenticatedLoaders.partnerLoader
+    )
+
+    const partnerLoader =
+      authenticatedLoaders?.partnerLoader ??
+      unauthenticatedLoaders.partnerLoader
+
```

Comment:
A little spooky ^, 

Currently, testing and it looks like if you pass in no auth information you'll get the unauthed data which is great but will this stop you if youre another partner's user? (ex: Im Gagosian user trying to access data for Commerce Test)

Need to test that but hopefully Gravity API layer will catch things like that as well!

Reply:
MP just passes through, Gravity is the ultimate source of authorization truth. So as long as gravity behaves properly, there shouldn't be any bug in MP.

Reply:
ie - if you're logged-in as a user with access to Gagosian, due to including a token you'll get the auth'ed loader - no caching - and Gravity would 401/403 is you tried to access a different partner.

Reply:
Welp I see what you mean regarding adding a new type thats CMS Partner only -- was thinking this could be a nice stepping stone to that worlds but had to touch a lot of the test setup for this fork. 😬 not sure I like it